pkg/cloud/azure/actuators/machine/reconciler.go

```diff
@@ -136,13 +135,6 @@ func (s *Reconciler) Update(ctx context.Context) error {
 		return errors.New("returned incorrect vm interface")
 	}
 
-	// We can now compare the various Azure state to the state we were passed.
-	// We will check immutable state first, in order to fail quickly before
-	// moving on to state that we can mutate.
-	if isMachineOutdated(s.scope.MachineConfig, converters.SDKToVM(vm)) {
```

Comment:
is `SDKToVM` used somewhere else? if not can we please drop it?

Reply:
you're right